src/UNL/UCBCN/Manager/DeleteRecurrence.php

```diff
@@ -23,6 +24,11 @@ public function __construct($options = array())
             throw new \Exception("That calendar could not be found.", 404);
         }
 
+        $user = Auth::getCurrentUser();
+        if (!$user->hasPermission(Permission::EVENT_EDIT_ID, $this->calendar->id)) {
+            throw new \Exception("You do not have permission to edit events on this calendar.", 403);
```

Comment:
That should be edit because that is removing a single recurrence from a single datetime of an event.